sandbox/tilted_bbl.jl

```diff
@@ -0,0 +1,114 @@
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.TurbulenceClosures: AnisotropicMinimumDissipation
+using CUDA
+
+const Nx = 450; const Lx = 1000
+const Nz = 64; const Lz = 100
+const θ_rad = 0.05 # radians
+const θ_deg = rad2deg(θ_rad) # degrees
+const N²∞ = 1e-5
+const V∞ = 0.1
+const g̃ = (sin(θ_rad), 0, cos(θ_rad))
+
+#++++ Grid
+topo = (Periodic, Periodic, Bounded)
+
+S = 1.3
+zF(k) = Lz*(1 + tanh(S * ( (k - 1) / Nz - 1)) / tanh(S))
+grid = VerticallyStretchedRectilinearGrid(topology=topo,
+                                          architecture = CUDA.has_cuda() ? GPU() : CPU(), 
+                                          size=(Nx, 1, Nz), 
+                                          x=(0, Lx), y=(0, 6*Lx/Nx), zF=zF,
+                                          halo=(3,1,3),
+                                         )
+println(); println(grid); println()
+#----
+
+#++++ Buoyancy model and background
+buoyancy = Buoyancy(model=BuoyancyTracer(), gravitational_unit_vector=g̃)
+tracers = :b
+
+b∞(x, y, z, t) = N²∞ * (x*g̃[1] + z*g̃[3])
+B_field = BackgroundField(b∞)
+
+b_bottom(x, y, t) = -b∞(x, y, 0, t)
+grad_bc_b = GradientBoundaryCondition(b_bottom)
+b_bcs = TracerBoundaryConditions(grid, bottom = grad_bc_b)
+#----
+
+#+++++ Boundary Conditions
+#+++++ Bottom Drag
+const z₀ = 0.01 # roughness length (m)
```

Comment:
Would you want to align the equal signs and align the comments, just to pretty things up a little more?